src/ResourceManager/Compute/Stack/Commands.Compute.Test/ScenarioTests/DiagnosticsExtensionTests.cs

```diff
@@ -12,37 +12,21 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.Compute.Common;
 using Microsoft.WindowsAzure.Commands.ScenarioTest;
-using System;
-using System.IO;
-using System.Reflection;
 using Xunit;
 
 namespace Microsoft.Azure.Commands.Compute.Test.ScenarioTests
 {
     public class DiagnosticsExtensionTests
     {
-        private static string configDirPath = Path.Combine(Path.GetDirectoryName(Uri.UnescapeDataString(new Uri(Assembly.GetExecutingAssembly().CodeBase).AbsolutePath)), "ConfigFiles");
-
-        private static string GetConfigFilePath(string filename)
-        {
-            return Path.Combine(configDirPath, filename);
-        }
-
-        public DiagnosticsExtensionTests(Xunit.Abstractions.ITestOutputHelper output)
-        {
-            ServiceManagemenet.Common.Models.XunitTracingInterceptor.AddToContext(new ServiceManagemenet.Common.Models.XunitTracingInterceptor(output));
-        }
-
-        [Fact]
+        [Fact(Skip = "Not suported in Azure stack")]
```

Comment:
@deepaft we should remove all skipped tests in this PR if they are not supported in Stack

Reply:
Removed